normandy/recipes/filters.py

```diff
@@ -6,6 +6,18 @@
 Each filter below defines a rule, and when it matches a user. Filter parameters
 are represented as JSON. Most users will interact with filters at a higher
 level, such as a web interface that allows building a filter with a form.
+
+Filter objects are generally specified as a JSON object with at least a
+"type" field, and other fields determined by that type. For example,
+ChannelFilter` below has a type of "channel" and requires a ``channels``
```

Comment:
Should that be "ChannelFilter" with two backticks? ...since this gets included in Sphinx as rst.

Reply:
Eh, yeah this isn't right. I had originally tried to make a reference, but couldn't figure out so deleted the markdown. I apparently missed some though.